es-base-code/main/default/objects/Customer_Fields__mdt/fields/Customer_Reservation_Status_Value__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
@pozil This is a new custom metadata field that will hold the Reservation status value.